app/Metrics/App/WikiMetrics.php

```diff
@@ -14,15 +15,23 @@ public function saveMetrics(Wiki $wiki): void
         $oldRecord = WikiDailyMetrics::where('wiki_id', $wiki->id)->latest('date')->first();
         $todayPageCount = $wiki->wikiSiteStats()->first()->pages ?? 0;
         $isDeleted = (bool)$wiki->deleted_at;
+        $tripleCount = null;
+        if (!$isDeleted) {
+            $tripleCount = $this->getNumOfTriples($wiki);
+        }
+
         if ($oldRecord) {
             if ($oldRecord->is_deleted) {
-                \Log::info("Wiki is deleted, no new record for WikiMetrics ID {$wiki->id}.");
+                \Log::info("Wiki is deleted, no new record for Wiki ID {$wiki->id}.");
                 return;
             }
             if (!$isDeleted) {
                 if ($oldRecord->pages === $todayPageCount) {
-                    \Log::info("Page count unchanged for WikiMetrics ID {$wiki->id}, no new record added.");
-                    return;
+                    \Log::info("Page count unchanged for Wiki ID {$wiki->id}, no new record added.");
```

Comment:
oof, logic gets complicated here I think again but in this case a new record could still be added if the number of triples changes right?

I wonder if we now want to pull out all this logic into it's own method(s) to make it easier for humans to parse (and ensure we get the messages we send to users right)?